### DISCUSSION TITLE
Can I use one VPS server on several devices?

### DISCUSSION THREAD

**Discussion by Silvanskii** [Ysurac/openmptcprouter#2339] [Ideas]
I think to buy a server for more than 5 devices on 8 LTE modems, is it realistic?

**Comment by Ysurac** [comment]
No. Multiple devices on a VPS is not supported. There is a not documented mode for that but it's not really great and should only be used if lan2lan is needed.

**Reply by mstrolo** [reply]
why not?

so how do users with more than 1 client use one vps?

**Reply by Ysurac** [reply]
They have one VPS for each client.
It's a bad idea to have more than one client on a VPS for many reason:
* not possible to configure MPTCP for each client (one kernel => one setting)
* one ip is required by client else not possible to configure firewall by client
* VPS don't have so much bandwidth in most case
* VPS are cheap or are easy to create on a server, why doing something bad in this case ?
*  ....

**Reply by mstrolo** [reply]
so what does every client need like there on a server with 1 CPU and 1 ram?
with the best bandwidth?

**Reply by mstrolo** [reply]
ok so i have a server with mptcp kernel and I installed Docker on it.

i have a container and it shows that mptcp enabled how do I install the vps?
